templates/referral.html

```diff
@@ -161,29 +161,23 @@
                                     <!-- COPY -->
                                     <table width="100%" border="0" cellspacing="0" cellpadding="0">
                                         <tr>
-                                            <td align="center" style="font-size: 25px; font-family: Helvetica, Arial, sans-serif; color: #333333; padding-top: 30px;" class="padding-copy">You've been invited to try Storj!</td>
+                                          <td align="center" style="font-size: 25px; font-family: Helvetica, Arial, sans-serif; color: #333333; padding-top: 30px;" class="padding-copy">You've been invited to try Storj!</td>
                                         </tr>
                                         <tr>
                                             <td align="center" style="padding: 20px 0 0 0; font-size: 16px; line-height: 25px; font-family: Helvetica, Arial, sans-serif; color: #777777;" class="padding-copy">
-                                                <p>
-                                                    <p>
-                                                        {{senderEmail}} sent you an invite,
-                                                    </p>
+                                                    <p>Great news! Your friend {{senderEmail}} just sent you 50GB of free storage and 50GB of free bandwidth on Storj.</p>
 
-                                                    <p>
-                                                        Sign up at the link below to receive your $10 free credit.
-                                                    </p>
+                                                    <p>Storj is a decentralized cloud storage platform. 
+                                                        It’s a blazing fast, highly secure and cost effective alternative to regular providers, with a foundation in P2P technology and the blockchain.</p>
 
-                                                    <p>
-                                                        {{#if url}}
-                                                            <a href="{{url}}">{{url}}</a>
-                                                        {{/if}}
-                                                    </p>
+
+                                                    <p>Claim your free credits today and experience the cloud in a new way.</p>
+
+                                                    <p>Sign up below and get 50GB of storage and 50GB of transfer per month (for the next three months)-- on us!</p>
```

Comment:
I told you wrong. Should be 'get 50GB of storage and 50GB of bandwidth'. Bandwidth, not transfer.